src/containers/Layout/index.tsx

```diff
@@ -1,13 +1,16 @@
-import { Outlet } from 'react-router-dom';
-
+import { ReactNode } from 'react';
 import { Footer } from '~/containers/Footer';
 import { Header } from '~/containers/Header';
```

Comment:
I think you can group those imports
```suggestion
import { Header, Footer } from '~/containers';
```

Reply:
https://github.com/defi-wonderland/web3-react-boilerplate/commit/9b5c51f0ad472ae9a2584c0f0d364b6e6f83e685